### PR TITLE
Mergeback 2.0.0 to develop

### PR DESCRIPTION
**Pull request by yunchu**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have ran e2e tests and there is no issues.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

**Comment by harimkang** [comment]
@sungchul2 Can you please confirm if this PR is OK or not with some scale related changes on the detection side?
cc. @wonjuleee

File: src/otx/recipe/detection/atss_mobilenetv2.yaml [hunk 1]
```diff
@@ -58,8 +58,8 @@ overrides:
           - class_path: otx.core.data.transform_libs.torchvision.Resize
             init_args:
               scale:
-                - 736
                 - 992
+                - 800
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_mobilenetv2.yaml]:
```suggestion
                - 800
                - 992
```

File: src/otx/recipe/detection/atss_mobilenetv2.yaml [hunk 2]
```diff
@@ -83,8 +83,8 @@ overrides:
...
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_mobilenetv2.yaml]:
```suggestion
                - 800
                - 992
```

File: src/otx/recipe/detection/atss_mobilenetv2.yaml [hunk 3]
```diff
@@ -103,8 +103,8 @@ overrides:
...
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_mobilenetv2.yaml]:
```suggestion
                - 800
                - 992
```

File: src/otx/recipe/detection/atss_mobilenetv2_tile.yaml [hunk 4]
```diff
@@ -55,8 +55,8 @@ overrides:
           - class_path: otx.core.data.transform_libs.torchvision.Resize
             init_args:
               scale:
-                - 736
                 - 992
+                - 800
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_mobilenetv2_tile.yaml]:
```suggestion
                - 800
                - 992
```

File: src/otx/recipe/detection/atss_mobilenetv2_tile.yaml [hunk 5]
```diff
@@ -80,8 +80,8 @@ overrides:
...
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_mobilenetv2_tile.yaml]:
```suggestion
                - 800
                - 992
```

File: src/otx/recipe/detection/atss_mobilenetv2_tile.yaml [hunk 6]
```diff
@@ -100,8 +100,8 @@ overrides:
...
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_mobilenetv2_tile.yaml]:
```suggestion
                - 800
                - 992
```

File: src/otx/recipe/detection/atss_resnext101.yaml [hunk 7]
```diff
@@ -58,8 +58,8 @@ overrides:
           - class_path: otx.core.data.transform_libs.torchvision.Resize
             init_args:
               scale:
-                - 736
                 - 992
+                - 800
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_resnext101.yaml]:
```suggestion
                - 800
                - 992
```

File: src/otx/recipe/detection/atss_resnext101.yaml [hunk 8]
```diff
@@ -83,8 +83,8 @@ overrides:
...
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_resnext101.yaml]:
```suggestion
                - 800
                - 992
```

File: src/otx/recipe/detection/atss_resnext101.yaml [hunk 9]
```diff
@@ -103,8 +103,8 @@ overrides:
...
```
Comment (sungchul1) [inline src/otx/recipe/detection/atss_resnext101.yaml]:
```suggestion
                - 800
                - 992
```

File: tests/unit/engine/utils/test_auto_configurator.py [hunk 10]
```diff
@@ -171,7 +171,7 @@ def test_update_ov_subset_pipeline(self) -> None:
             {
                 "class_path": "otx.core.data.transform_libs.torchvision.Resize",
                 "init_args": {
-                    "scale": [736, 992],
+                    "scale": [992, 800],
```
Comment (sungchul1) [inline tests/unit/engine/utils/test_auto_configurator.py]:
```suggestion
                    "scale": [800, 992],
```

**Review by sungchul1 — COMMENTED** [review]
For detection task in develop, the order of height-width should be changed.

**Comment by yunchu** [comment]
@sungchul2 i've pushed w/h order changes, pleases review once again. thanks

**Review by harimkang — DISMISSED** [review]

File: src/otx/recipe/detection/atss_mobilenetv2.yaml (hunk 1 above)
Comment (sungchul1) [reply src/otx/recipe/detection/atss_mobilenetv2.yaml]: @yunchu It seems not reflected, please check it.

**Review by chuneuny-emily — DISMISSED** [review]

File: src/otx/recipe/detection/atss_mobilenetv2.yaml (hunk 1 above)
Comment (yunchu) [reply src/otx/recipe/detection/atss_mobilenetv2.yaml]: updated

**Review by sungchul1 — DISMISSED** [review]

**Review by harimkang — DISMISSED** [review]

**Review by harimkang — APPROVED** [review]

**Review by sungchul1 — APPROVED** [review]

**Pull request merged**